cpp/src/Glacier2Lib/SessionHelper.cpp

```diff
@@ -495,14 +497,9 @@ SessionHelperI::connected(const Glacier2::RouterPrx& router, const optional<Glac
             _session = session;
             _connected = true;
 
-            if (acmTimeout > 0)
```

Comment:
It's not clear to me why the close callback was sent only for acmTimeout > 0.

Reply:
I also don't know why we do this.